### PR TITLE
Update harsha.md

### PR DESCRIPTION
**Pull request by Harsha-Ambati**

### CODE REVIEW

**Review by pranavgoel29 — APPROVED** [review]
Good to go 👍🏻.

File: firstpr/harsha.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+  <img align="left" alt="Ajay's Github" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/github.svg" />
+</a>
+<a href="https://t.me/Bugbounty1">
+  <img align="left" alt="Ajay's Telegram" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/telegram.svg" />
+</a>
+</br>
+
+### I am Harsha
+
+- An Cyber security enthusaist
```
Comment (arthurbuhl) [inline firstpr/harsha.md:20]: You have a typo here. Its `enthusiast` actually, not `enthusaist`

File: firstpr/harsha.md [hunk 2]
```diff
@@ -1 +1,26 @@
-##### Harsha Vardhan from india . B.tech student
+## Hello World! 
+<img src="https://raw.githubusercontent.com/iampavangandhi/iampavangandhi/master/gifs/Hi.gif" width="30px"></h2>
+
+<a href="https://twitter.com/harsha0x01">
+  <img align="left" alt="Ajay's Twitter" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/twitter.svg" />
+</a>
+<a href="https://www.linkedin.com/in/harsha-ambati">
+  <img align="left" alt="Ajay's Linkdein" width="22px" src="https://cdn.jsdelivr.net/npm/simple-icons@v3/icons/linkedin.svg" />
```
Comment (arthurbuhl) [inline firstpr/harsha.md:8]: Also, here.. its `LinkedIn` not `Linkdein`

**Comment by Harsha-Ambati** [comment]
can i update it again!! or you already done!

**Comment by arthurbuhl** [comment]
Yes, you can always update your local repo and then git commit + push again and this PR will be auto-updated with your latest changes.

**Comment by proful** [comment]
Ask in Twitter group or discord by giving link to this pr for review and approval. 2 approval needed.

**Review by sarathkumar6 — APPROVED** [review]
Nice job.

**Pull request merged**